package.json

```diff
@@ -39,7 +39,7 @@
         "@fortawesome/free-solid-svg-icons": "^6.5.2",
         "@fortawesome/vue-fontawesome": "^3.0.8",
         "@geoblocks/cesium-compass": "^0.5.0",
-        "@geoblocks/mapfishprint": "^0.2.14",
+        "@geoblocks/mapfishprint": "github:geoblocks/mapfishprint#932253b0e8ccc0c1d911cab8e02c5b2fc7a061af",
```

Comment:
Would be much better to wait and have a proper release tag here to go to prod (merging this PR will push it directly to prod !)